api/environments/dynamodb/types.py

```diff
@@ -88,6 +88,7 @@ class IdentityOverrideV2(BaseModel):
     identifier: str
     identity_uuid: str
     feature_state: FeatureStateModel
+    more_identity_overrides: bool = False
```

Comment:
I didn't check thoroughly, but this added field might prompt the backend to index this field in DynamoDB! It doesn't belong here.

Reply:
I'm not 100% what you mean by 'index' here, but yes, I believe that adding this field here would mean that it gets written to each record. Hence, it would be included in the index that we have on that table which projects all attributes. 

I agree with Kim that this is not desirable for sure. 

If this approach simplifies the code though, we could just sub class `IndentityOverrideV2`, add this new field and use the sub class for whatever logic relies on the field.

Reply:
Another problem, which I pointed out [here](https://github.com/Flagsmith/flagsmith/pull/4840/files#r1868126098), is that this field doesn't belong to an individual override document.

Reply:
Got it, yep, agreed.

Reply:
Ok this has now been removed.